templates/template.py

```diff
@@ -218,6 +218,7 @@ def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
             data_dir = self.config.data_dir
 
         # Not all datasets have predefined canonical train/val/test splits. If your dataset does not have any splits, you can omit any missing splits.
+        # If your dataset has no predefined splits, use datasets.Split.TRAIN for all of the data.
```

Comment:
This seems a bit confusing when coupled with the previous line. I would remove "If your dataset does not have any splits, you can omit any missing splits."